docs/core/additional-tools/dotnet-core-uninstall.md

```diff
@@ -0,0 +1,335 @@
+
+* **`--all-but-latest`**
+
+  Remove .NET Core SDKs or Runtimes, except the one highest version.
+
+* **`--all-lower-patches`**
+
+  Remove .NET Core SDKs or Runtimes that have been superceded by higher patches.
+
+* **`--all-previews`**
+
+  Remove .NET Core SDKs or Runtimes that are marked as previews.
+
+* **`--all-previews-but-latest`**
+
+  Remove .NET Core SDKs or Runtimes that are marked as previews except the one highest preview.
+
+* **`--aspnet-runtime`**
+
+  Remove ASP.NET Core Runtimes only.
+
+* **`--dry-run`**
+
+  Display .NET Core SDKs and Runtimes that will be removed.
+
+* **`--hosting-bundle`**
+
+  Remove .NET Core Runtime & Hosting Bundles only.
+
+* **`--major-minor <MAJOR_MINOR>`**
+
+  Remove .NET Core SDKs or Runtimes that match the specified `major.minor` version.
+
+* **`--runtime`**
+
+  Remove .NET Core Runtimes only.
+
+* **`--sdk`**
+
+  Remove .NET Core SDKs only.
+
+* **`-v, --verbosity <LEVEL>`**
+
+  Set the verbosity level. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
+
+* **`--x64`**
+
+  Can be used with `--sdk`, `--runtime` and `--aspnet-runtime` to remove x64.
+
+* **`--x86`**
+
+  Can be used with `--sdk`, `--runtime` and `--aspnet-runtime` to remove x86.
+
+* **`-y, --yes`**
+  Execute the command without requiring Y/n confirmation.
+
+Note:
+1. Exactly one of `--sdk` and `--runtime` is required.
+2. `--all`, `--all-below`, `--all-but`, `--all-but-latest`, `--all-lower-patches`, `--all-previews`, `--all-previews-but-latest`, `--major-minor` and `[<VERSION>...]` are exclusive.
+3. If neither of `--x64` or `--x86` is specified, then both x64 and x86 will be removed.
+
+---
+
+#### Examples
+
+* Dry run of uninstalling all .NET Core Runtimes that have been superceded by higher patches.
+
+  ```
+  dotnet-core-uninstall --all-lower-patches --runtime --dry-run
+  ```
+
+* Dry run of uninstalling all .NET Core SDKs below the version `2.2.301`:
+
+  ```
+  dotnet-core-uninstall --all-below 2.2.301 --sdk --dry-run
+  ```
+
+* Uninstall all x86 .NET Core Runtimes except the version `3.0.0-preview6-27804-01` without requiring Y/n confirmation:
+
+  ```
+  dotnet-core-uninstall --all-but 3.0.0-preview6-27804-01 --runtime --yes
+  ```
+
+* Uninstall all .NET Core 1.1 SDKs without requiring Y/n confirmation:
+
+  ```
+  dotnet-core-uninstall --major-minor 1.1 -y
+  ```
+
+* Uninstall all .NET Core SDKs that can be removed by this tool:
+
+  ```
+  dotnet-core-uninstall --all --sdk
+  ```
+
+### Optional: deleting the `NuGetFallbackFolder` folder
+
+The `NuGetFallbackFolder` folder contains a cache of NuGet packages used by a .NET Core SDK during the restore operation, such as running `dotnet restore` or `dotnet build /t:Restore`. It is located at `C:\Program Files\dotnet\sdk` on Windows and at `/usr/local/share/dotnet/sdk` on macOS. Delete this folder from your machine if you no longer need it after uninstalling .NET Core SDKs.
```

Comment:
Suggest moving your text here to docs/core/versions/remove-runtime-sdk-versions.md  and changing this to

```suggestion
In some cases, you no longer need the NuGetFallbackFolder and may wish to delete it. See [[ section reference to docs/core/versions/remove-runtime-sdk-versions.md ]]
```